model/dataProvider/providers/LtiUserDataProviderService.php

```diff
@@ -0,0 +1,63 @@
+<?php
+/**
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; under version 2
+ * of the License (non-upgradable).
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ *
+ * Copyright (c) 2019  (original work) Open Assessment Technologies SA;
+ */
+
+namespace oat\taoSyncClient\model\dataProvider\providers;
+
+use oat\oatbox\service\ConfigurableService;
+use oat\taoEncryption\Service\Lti\LaunchData\EncryptedLtiLaunchDataStorage;
```

Comment:
we don't need this dependency on the taoEncryptionService, we need to get rid of this extension and pick data from the resources only. (You can leave it as it is, but add here `@TODO try to rewrite without taoEncryptionService`)